frontend/pages/dashboard/new-post.tsx

```diff
@@ -87,6 +139,35 @@ const NewPostPage: NextPage = () => {
             onChange={setLangId}
           />
 
+          <div className="header-preview-container">
+            <PostHeader
+              postTitle={title}
+              postStatus={PostStatusType.Published}
+              publishDate={new Date().toISOString()}
+              authorName={currentUser?.name || 'anonymous'}
```

Comment:
When would we see the `anonymous` value?

Reply:
We'll never see it, this was mostly just to satisfy TypeScript 😊